mayhem/FuzzedDataProvider.h

```diff
@@ -0,0 +1,397 @@
+//===- FuzzedDataProvider.h - Utility header for fuzz targets ---*- C++ -* ===//
```

Comment:
Prefer not to include FDP. Either pull the precompiled clang from LLVM or install from source.